usage_report/utils/s3_utils.py

```diff
@@ -1,56 +1,52 @@
-import boto
+import boto3
 import json
-from boto.s3.key import Key
 
 
-def filename_from_date(filename, date):
-    chunks = filename.split('.')
-    return chunks[0] + '_' + date + '.' + '.'.join(chunks[1:])
+def create_session(aws_access_key_id=None, aws_secret_access_key=None):
```

Comment:
This function seems unnecessary, since it is only directly calling `boto3.Session`.